cmd/goal/README.md

```diff
@@ -0,0 +1,151 @@
+# Example `goal` Snippets
+
+Unless otherwise noted, it is assumed that the working directory
+begins at the top level of the `go-algorand` repo.
+
+## Starting a Single Node Dev Network 
+
+### Q:
+> Having just done a new build in go-algorand, how do I get a single node dev network up, with algos in an easily used wallet from goal?
+
+### A:
+
+```sh
+# create a networks directory if you don't already have it: 
+mkdir -p ~/networks
+
+# Set this to where you want to keep the network files (and data dirs will go beneath)
+NETWORKS=~/networks
+
+# set this to "name" your network:
+NAME=niftynetwork
+
+goal network create -n $NAME -r $NETWORKS/$NAME -t ./test/testdata/nettemplates/OneNodeFuture.json
```

Comment:
Not sure if this is pedantic, but assuming the user used `make install` like in the readme, it might be worth noting explicitly that `${GOPATH}/bin/goal` should be used here.

https://github.com/algorand/go-algorand/blob/b53a6da7e2c0d72edf9b3940a7f3057f93a086ec/README.md?plain=1#L76

Reply:
I've added commentary to the README